package.json

```diff
@@ -130,7 +131,6 @@
     "@types/minimatch": "^3.0.3",
     "@types/node": "18.0.0",
     "@types/pdfjs-dist": "^2.10.378",
-    "@types/request": "^2.48.9",
```

Comment:
You removed `request types` but the `request` library is still there so I guess you should remove it as well. Also I think you should run `npm i` afterwards to regenerate `package-lock.json`